src/VDPRegViewer.cpp

```diff
@@ -905,17 +912,26 @@ void VDPRegViewer::connectHighLights()
 	makeGroup(list, label_dec_r26);
 }
 
+static constexpr unsigned TOTAL_SIZE = 64 + 16 + 2 + 3;
```

Comment:
For now, this is only used inside `VDPRegViewer::refresh()`, right? Then maybe move this inside that method.

To avoid repeating the same expressions, it's probably better to define this as `TOTAL_SIZE = sizeof(regs)`.
(Or alternatively, make `regs` a `std::array`, and then do `TOTAL_SIZE = regs.size()`.)